src/controller/controller.c

```diff
@@ -940,6 +940,48 @@ static int controller_property_get_status(
         return sd_bus_message_append(reply, "s", controller_get_system_status(controller));
 }
 
+static int controller_property_get_loglevel(
+                UNUSED sd_bus *bus,
+                UNUSED const char *path,
+                UNUSED const char *interface,
+                UNUSED const char *property,
+                sd_bus_message *reply,
+                UNUSED void *userdata,
+                UNUSED sd_bus_error *ret_error) {
+        const char *log_level = log_level_to_string(bc_log_get_level());
+        return sd_bus_message_append(reply, "s", log_level);
+}
+
+static int controller_property_get_log_target(
+                UNUSED sd_bus *bus,
+                UNUSED const char *path,
+                UNUSED const char *interface,
+                UNUSED const char *property,
+                sd_bus_message *reply,
+                UNUSED void *userdata,
+                UNUSED sd_bus_error *ret_error) {
+
+        LogFn logfn = bc_log_get_log_fn();
+        const char *journald_str = "journald";
+        const char *stderr_full_str = "stderr_full";
+        const char *stderr_str = "stderr";
```

Comment:
Shouldn't we be using constants from https://github.com/eclipse-bluechi/bluechi/blob/main/src/libbluechi/log/log.h#L10 ?

Reply:
Yes, I agree. And in addition, it would be great to move this (line 964 to 980) into a dedicated function, e.g. `const char* log_target_to_str(LogFn logfn)` and reuse it here and in `agent.c`.